README.md

```diff
@@ -1,3 +1,78 @@
-## JavaScript Style Guide
+# JavaScript Style Guide
 
 An opinionated and adequately reasonable guide to writing JavaScript
+
+## Naming
+
+### Files
+
+#### PascalCase
+
+- Component file names
+- If a folder that functions as a component (i.e. contains an index.js which exports a component)
+- Stories files (`.stories.js`)
+- Test files of components (`.test.js`, `.spec.js`)
```

Comment:
Utility type files should generally be camelCased. Essentially, however the export of file is going to be consumed should define the casing of the file name.

Take the component `SignOut` and the utility function `signOut`. The component should be in a file named `SignOut.js` and the function in one named `signOut.js`. (Could be a terrible example, because those two should not live in the same folder with that name, etc.)